Tabloid/client/src/components/categories/CategoryDelete.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { getById, deleteCategory } from "../../Managers/CategoryManager";
+
+const CategoryDelete = () => {
+    const [chosenCategory, setChosenCategory] = useState({});
+
+    const navigate = useNavigate();
+    const {id} = useParams();
+
+
+    useEffect(
+    () => {
+        getById(id).then((c) => {setChosenCategory(c)}).then(console.log(chosenCategory))
+        
+    },
+    []
+    )
+
+    console.log(chosenCategory)
+
+    
+    const Delete = () => {
+        deleteCategory(chosenCategory.id).then((c) => {
+            navigate('/categories')
+        })
+    }
+    const Cancel = () => {
+        navigate('/categories')
+    }
+
+    return (
+        <div style={{display:'flex', flexDirection: 'column', letterSpacing: '.5px', alignItems: 'center', margin: '45px', height: '30px', width: '500px', justifyContent: 'space-between'}}>
+            <h5 style={{marginBottom: '45px'}}>Are you sure you wish to delete the category?</h5>
+            <div style={{display: 'flex'}}>
+            <h5 style={{ marginRight: '30px' }}>{chosenCategory.name}</h5>
```

Comment:
chosenCategory.name keeps showing up as "Technology" for each category. On the page and in the components tab

Reply:
Thanks Cara and Tiffany for the help